.circleci/config.yml

```diff
@@ -92,7 +92,22 @@ jobs:
       - attach_workspace:
           at: ~/answers
       - run: ./.circleci/run_acceptance.sh
-        
+  # verify all translations are present
+  translation_test:
+    docker:
+      - image: circleci/node:14.5
+    working_directory: ~/answers
+    steps:
+      - setup-workspace
+      - attach_workspace:
+          at: ~/answers
+      - run: 
+          name: Verify translation files
+          command: |
+            npm run extract-translations
+            sudo apt-get install gettext
```

Comment:
maybe we could add a quiet flag (looks like this would be -qq) to make the output slightly quieter? it's not a big deal but it would make the output easier to read when there's an error

Reply:
sure